### PR TITLE
docs: Add image guidlines

### PR DESCRIPTION
**Pull request by salonigupta1**
# Checklist

- [ ] fixes #xxxx
- [ ] Passes `Semantic Pull Request` and `TypoCheck`
- [ ] New docs are listed in `README.md`
- [ ] New docs are listed in `index.md`
- [ ] Updated table of content
- [ ] Updated nav links in mkdocs.yml

---

## PR description

Write a brief note here what does your PR changes.

### CODE REVIEW

**Review by syedjawadakhtar — APPROVED** [review]
@pnijhara the rules are right?

File: docs/Development/CONTRIBUTING.md [hunk 1]
```diff
@@ -10,6 +10,14 @@ Please note we have a code of conduct, please follow it in all your interactions
 
 2.Update the `README.md` file with details of changes to the interface, this includes new environment variables, exposed ports, useful file locations and container parameters.
```
Comment (pnijhara) [inline docs/Development/CONTRIBUTING.md:11]: environment variables?
exposed ports?
Are we really using these stuff?

File: docs/Development/CONTRIBUTING.md [hunk 2]
```diff
@@ -10,6 +10,14 @@ Please note we have a code of conduct, please follow it in all your interactions
...
 
+## Guidelines for adding images:
+
+- All the images/figures/diagrams must always be added to a subfolder static. Make the folder if it isn't present
+- ScreenShots should be considered over camera photos.
+- The format of naming the images must be: <document_name>_. eg. online_CF_1.png for image in Online Crowd Funding.
```
Comment (pnijhara) [inline docs/Development/CONTRIBUTING.md]: Underline the example.

**Review by pnijhara — CHANGES_REQUESTED** [review]

**Comment by pnijhara** [comment]
Everything looks good! Nice work @salonigupta1.
I have suggested some changes please look at them. Also, fill in the PR template.

@syedjawadakhtar look at the commit message. You have to follow this kind of commit message.

File: docs/Development/CONTRIBUTING.md (hunk 1 above)
Comment (salonigupta1) [reply docs/Development/CONTRIBUTING.md:11]: so, should I delete this whole point or only change the stuff that you are indicating?

Comment (pnijhara) [reply docs/Development/CONTRIBUTING.md:11]: ask yourself this question!

Comment (pnijhara) [reply docs/Development/CONTRIBUTING.md:11]: Are you on it?

Comment (salonigupta1) [reply docs/Development/CONTRIBUTING.md:11]: yes

Comment (pnijhara) [reply docs/Development/CONTRIBUTING.md:11]: Complete it fast!
you are taking a lot of time on this fix.

**Comment by pnijhara** [comment]
@salonigupta1 could you please adjust these 3 points somewhere?

1. Never change a large of files (change iff required)
2. Never leave empty whitespaces after the EOL
3. Give 2 spaces instead of giving a tab.

**Comment by pnijhara** [comment]
@salonigupta1 are you on it?
Please finish it ASAP and ping me when you are done. I'll review this.

**Review by pnijhara — APPROVED** [review]

**Pull request merged**